actionview/test/template/template_error_test.rb

```diff
@@ -34,4 +34,20 @@ def test_provides_useful_inspect
 
     assert_equal "#<ActionView::Template::Error: original>", error.inspect
   end
+
+
+  def test_annotated_source_code_returns_empty_array_if_source_cant_be_found
+    template = Class.new do
+      def identifier
+        "something"
+      end
+    end.new
+
+    error = begin
+      raise Exception.new
+    rescue Exception
```

Comment:
Would it be clearer with just `raise` and then `rescue` without any specific exception?

Reply:
yes, yes it would!